guacamole-common-js/src/main/webapp/modules/Mouse.js

```diff
@@ -258,9 +258,15 @@ Guacamole.Mouse = function Mouse(element) {
 
     }
 
-    element.addEventListener('DOMMouseScroll', mousewheel_handler, false);
-    element.addEventListener('mousewheel',     mousewheel_handler, false);
-    element.addEventListener('wheel',          mousewheel_handler, false);
+    if (window.WheelEvent) {
+        // All modern browsers support wheel events.
+        element.addEventListener('wheel',          mousewheel_handler, false);
```

Comment:
Because these lines are now separated by `if/else` blocks and comments, I don't know that there's any reason to have all of the extra spaces between the `'wheel',` and the `mousewheel_handler` - both this line, and below.